packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2595,61 +2589,28 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         }
     }
 
-    private submitMaybeChunkedMessages(
+    private submitMessage(
```

Comment:
#nit there's no need for a separate function IMO. You can do this inside `submit`